shuup/default_reports/forms.py

```diff
@@ -45,6 +48,43 @@ def initial_contacts(self, key):
         return []
 
 
+class OrderLineReportForm(BaseReportForm):
+
+    order_line_type = forms.MultipleChoiceField(
+        label=_("Order Line Type"),
+        required=False,
+        choices=[
+            (line_type.value,
+             line_type.name.capitalize()) for line_type in OrderLineType],
+        initial=[1])  # Because value of OrderLineType.PRODUCT is 1
+
+    def __init__(self, *args, **kwargs):
+        super(OrderLineReportForm, self).__init__(*args, **kwargs)
+
+        supplier = Select2MultipleField(label=_("Suppliers"),
+                                        model=Supplier,
+                                        required=False,
+                                        help_text=_("Filter order lines by suppliers."))
+        order_status = Select2MultipleField(label=_("Order status"),
```

Comment:
I guess here you could just use `MultipleModelChoiceField` as I doubt we would have more than 10 different statuses and it is better for the merchant to see the list of status instead of trying to find one they want.